specification/validated-relationships-service-api.yaml

```diff
@@ -149,6 +149,37 @@ servers:
   - url: 'https://api.service.nhs.uk/validated-relationships/FHIR/R4'
     description: Production environment.
 paths:
+  /QuestionnaireResponse:
+    post:
+      summary: New Access request
+      description: |
+        ## Overview
+        For any new access request, the necessary details should be collected from a user facing service e.g. 
+        Proxy Access Service and submitted as a QuestionaireResponse. 
+        
+        For the most part demographics information doesn't need to be provided in the access request since it can be pulled from PDS.
```

Comment:
Does this need a section about the sandbox try it now like `/RelatedPerson` or should it be removed from `/RelatedPerson` and the comment moved to a common area

Reply:
Will add it once it is added to the sandbox